src/main/java/de/tum/in/www1/exerciseapp/domain/SubmittedAnswer.java

```diff
@@ -126,4 +126,11 @@ public String toString() {
             ", scoreInPoints='" + getScoreInPoints() + "'" +
             "}";
     }
+
+    /**
+     * Delete all references to question and question-elements if the question was changed
+     *
+     * @param quizExercise the changed quizExercise-object
+     */
+    public abstract void updateForDeletedReferences (QuizExercise quizExercise);
```

Comment:
same comment as above, please use a different method name of you actually delete something --> If I read "update", I would assume that nothing is deleted